### PR TITLE
[MNT] Add dependabot.yml

### LINKED ISSUE #90: Add dependabot.yml
See https://github.com/neurobagel/old-query-tool/blob/main/.github/dependabot.yml from Vue implementation of query tool for reference.

### PR DESCRIPTION
**Pull request by Sauradip07**
- Closes #90 

Changes proposed in this pull request:

- Added ` dependabot.yml ` for  automatically check and updates to dependencies

## Checklist

_This section is for the PR reviewer_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING#pull-request-guidelines) for more info)_
- [ ] PR has a label for the release changelog or `skip-release` (to be applied by maintainers only)
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:

- [ ] Tests have been added

For bug fixes:

- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+    ignore:
+      - dependency-name: 'react'
+        update-types: ['version-update:semver-major']
+      - dependency-name: 'typescript'
+        update-types: ['version-update:semver-major']
```
Comment (rmanaem) [inline .github/dependabot.yml]: We'd like to get updates for these dependencies so let's remove the `ignore`.

**Review by rmanaem — CHANGES_REQUESTED** [review]
Thanks for the PR @Sauradip07!
Please see the requested change.

**Review by rmanaem — APPROVED** [review]
🧑‍🍳

**Comment by surchs** [comment]
:rocket: PR was released in [`v0.2.0`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.0) :rocket:

**Pull request merged**